### PR TITLE
chore: remove published clients

### PR DESCRIPTION
**Pull request by FrankEssenberger**
First PR to remove a reference to the published clients.

### CODE REVIEW

File: samples/cf-sample-application/README.md [hunk 1]
```diff
@@ -45,6 +45,16 @@ cd cloud-sdk-js
...
 
+### Create oData Client
+
+Due to intellectual property reasons we are not allowed to include SAP service definitions in this open source repository.
+Please download the following service definitions  in `EDMX` format from the API hub:
+- [Business Partner service onPremise](https://api.sap.com/api/OP_API_BUSINESS_PARTNER_SRV/overview)
+- [Business Partner service cloud](https://api.sap.com/api/API_BUSINESS_PARTNER/overview)
+
+Store the two files as `cloud-business-partner-service.edmx` and `op-business-partner-service.edmx` in the folder `resources/service-specs`.
+Generate the client by executing the npm command `npm run generate-client`
```
Comment (tomfrenken) [inline samples/cf-sample-application/README.md:56]: Add `run npm install` before

File: samples/cf-sample-application/README.md [hunk 2]
```diff
@@ -45,6 +45,16 @@ cd cloud-sdk-js
 git sparse-checkout set samples/cf-sample-application
 ```
 
+### Create oData Client
+
+Due to intellectual property reasons we are not allowed to include SAP service definitions in this open source repository.
```
Comment (fwilhe) [inline samples/cf-sample-application/README.md:50]: This phrasing seems a bit odd to me.

Maybe something like "For legal reasons, this repository can't contain the SAP service definitions."

File: samples/helm-sample-application/application/src/cloud-business-partner/cloud-business-partner.service.ts [hunk 3]
```diff
@@ -1,18 +1,19 @@
 import { Injectable } from '@nestjs/common';
 import {
   BusinessPartner,
-  businessPartnerService,
-} from '@sap/cloud-sdk-vdm-business-partner-service';
+  cloudBusinessPartnerService as businessPartnerService,
+} from '../generated/cloud-business-partner-service';
 const { businessPartnerApi } = businessPartnerService();
 
-const destinationName: string = process.env.CLOUD_DESTINATION;
```
Comment (tomfrenken) [inline samples/helm-sample-application/application/src/cloud-business-partner/cloud-business-partner.service.ts:8]: I believe this was accidentally removed while the code was copied between samples?
This is helm-specific logic, so this part has to be reverted.

File: samples/helm-sample-application/application/src/cloud-business-partner/cloud-business-partner.service.ts [hunk 4]
```diff
@@ -1,18 +1,19 @@
...
 @Injectable()
 export class CloudBusinessPartnerService {
   async getFiveBusinessPartners(): Promise<BusinessPartner[]> {
     return businessPartnerApi
       .requestBuilder()
       .getAll()
       .top(5)
-      .execute({ destinationName: destinationName });
```
Comment (tomfrenken) [inline samples/helm-sample-application/application/src/cloud-business-partner/cloud-business-partner.service.ts:16]: Same here & on the other 2 endpoints

File: samples/helm-sample-application/application/README.md [hunk 5]
```diff
@@ -51,6 +51,16 @@ cd cloud-sdk-js
...
 
+### Create oData Client
+
+For legal reasons, this repository must not contain the SAP service definitions.
+Please download the following service definitions  in `EDMX` format from the API hub:
+- [Business Partner service onPremise](https://api.sap.com/api/OP_API_BUSINESS_PARTNER_SRV/overview)
+- [Business Partner service cloud](https://api.sap.com/api/API_BUSINESS_PARTNER/overview)
+
+Store the two files as `cloud-business-partner-service.edmx` and `op-business-partner-service.edmx` in the folder `resources/service-specs`.
+Run `npm i` and generate the client by executing the npm command `npm run generate-client`
```
Comment (tomfrenken) [inline samples/helm-sample-application/application/README.md:62]:
```suggestion
Run `npm i` and generate the client by executing the npm command `npm run generate-client`.
```

File: samples/k8s-sample-application/README.md [hunk 6]
```diff
@@ -50,6 +50,16 @@ cd cloud-sdk-js
...
 
+### Create oData Client
+
+For legal reasons, this repository must not contain the SAP service definitions.
+Please download the following service definitions  in `EDMX` format from the API hub:
+- [Business Partner service onPremise](https://api.sap.com/api/OP_API_BUSINESS_PARTNER_SRV/overview)
+- [Business Partner service cloud](https://api.sap.com/api/API_BUSINESS_PARTNER/overview)
+
+Store the two files as `cloud-business-partner-service.edmx` and `op-business-partner-service.edmx` in the folder `resources/service-specs`.
+Run `npm i` and generate the client by executing the npm command `npm run generate-client`
```
Comment (tomfrenken) [inline samples/k8s-sample-application/README.md:61]:
```suggestion
Run `npm i` and generate the client by executing the npm command `npm run generate-client`.
```

File: samples/cf-sample-application/README.md [hunk 7]
```diff
@@ -45,6 +45,16 @@ cd cloud-sdk-js
...
 
+### Create oData Client
+
+For legal reasons, this repository must not contain the SAP service definitions.
+Please download the following service definitions  in `EDMX` format from the API hub:
+- [Business Partner service onPremise](https://api.sap.com/api/OP_API_BUSINESS_PARTNER_SRV/overview)
+- [Business Partner service cloud](https://api.sap.com/api/API_BUSINESS_PARTNER/overview)
+
+Store the two files as `cloud-business-partner-service.edmx` and `op-business-partner-service.edmx` in the folder `resources/service-specs`.
+Run `npm i` and generate the client by executing the npm command `npm run generate-client`
```
Comment (tomfrenken) [inline samples/cf-sample-application/README.md:56]:
```suggestion
Run `npm i` and generate the client by executing the npm command `npm run generate-client`.
```

**Review by tomfrenken — APPROVED** [review]
Overall LGTM, however, consider my comments in the helm sample (and some minor punctuation consistency comments)

File: samples/helm-sample-application/application/src/cloud-business-partner/cloud-business-partner.service.ts (hunk 3 above)
Comment (FrankEssenberger) [reply samples/helm-sample-application/application/src/cloud-business-partner/cloud-business-partner.service.ts:8]: ups yes good catch.

File: samples/helm-sample-application/application/src/cloud-business-partner/cloud-business-partner.service.ts (hunk 4 above)
Comment (FrankEssenberger) [reply samples/helm-sample-application/application/src/cloud-business-partner/cloud-business-partner.service.ts]: yes I have also one more look in the diff for suspicious things.

**Pull request merged**